backend/api/routers/contests.py

```diff
@@ -58,3 +58,44 @@ async def get_contests(
         raise HTTPException(status_code=404, detail=f"{id=} not found")
 
     return {'data': json.loads(json_util.dumps(data))}
+
+
+@router.delete(
+    '/',
+    responses={404: {'description': 'Contest with given id not found'}}
+)
+async def delete_contest(
+    request: Request,
+    id: str
+):
+    db = request.app.database
+
+    data = await db.contests.delete_one({'_id': ObjectId(id)})
+
+    if not data:
+        raise HTTPException(status_code=404, detail=f"{id=} not found")
+
```

Comment:
a to nie powinno za przeproszeniem usuwać też wszystkich zgłoszeń w tym konkursie i usuwać też wszystkie pliki (ze wszystkich zgłoszeń i też te terms and conditions i background)

Reply:
Oj tak

Reply:
O cholerka może tak być, tylko @AdamJ77 musiałby to obsłużyć w static files

Reply:
Tyle zmian ostatnio tam robiłeś, że możesz kontynuować @kozlowskimaciej

Reply:
Ale zajmę się tym